js/cs/main.coffee

```diff
@@ -458,6 +458,12 @@ app.controller 'TimeLord', ($scope, $http, $routeParams, $location, md5) ->
         $scope.modalState 'range_modal', false
       else if $scope.loginOpen == true
         $scope.modalState 'loginOpen', false
+      # The nature of using none Anular based event means that they won't be ingested into the Angular lifecycle.
```

Comment:
Should it be non-Angular?

Reply:
Yeah... Ich sprechehhhh nicht gutes english.